src/main/webapp/definitions/alfresco-model.yaml

```diff
@@ -0,0 +1,318 @@
+        ```JSON
+        {
+          "list": {
+            "pagination": {
+              "count": 0,
+              "hasMoreItems": true,
+              "totalItems": 0,
+              "skipCount": 0,
+              "maxItems": 0
+            },
+            "entries": [
+              {
+                "entry": {
+                  "name": "titled",
+                  "prefixedName": "cm:titled",
+                  "description": "Titled",
+                  "title": "Titled",
+                  "properties": [
+                    {
+                      "name": "title",
+                      "prefixedName": "cm:title",
+                      "title": "Title",
+                      "description": "Content Title",
+                      "dataType": "d:mltext",
+                      "facetable": "UNSET",
+                      "indexTokenisationMode": "BOTH",
+                      "multiValued": false,
+                      "mandatoryEnforced": false,
+                      "mandatory": false,
+                      "indexed": true
+                    },
+                    {
+                      "name": "description",
+                      "prefixedName": "cm:description",
+                      "title": "Description",
+                      "description": "Content Description",
+                      "dataType": "d:mltext",
+                      "facetable": "UNSET",
+                      "indexTokenisationMode": "BOTH",
+                      "multiValued": false,
+                      "mandatoryEnforced": false,
+                      "mandatory": false,
+                      "indexed": true
+                    }
+                  ]
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+              {
+                "entry": {
+                  ...
+                }
+              },
+            ]
+          }
+        }
+        ```
+      operationId: listAspects
+      parameters:
+        - name: where
+          in: query
+          description: |
+            Optionally filter the list. Here are some examples:
+
+            The following where clause will only return aspects from the customModel1 and customModel2.
+              ```
+              where=(modelIds=customModel1,customModel2)
+              ```
+
+              The following where clause will only return all the aspect from custom models.
+              ```
+              where=(customModelOnly=true)
+              ```
+          required: false
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/AspectPaging'
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/aspect/{aspectId}':
```

Comment:
This should be /aspects i.e. plural

Reply:
And I assume /models in your previous comment above (instead of /model) ?

Reply:
No, we're avoiding the /models endpoint for now.

/aspects will return ALL aspects in the system and /types will return ALL types (with filtering by model if necessary using the where clause). 

I also suggested we setup a short design meeting to discuss all this before we got too far with implementation to avoid too much to'ing and fro'ing in PR comments ;-)